home/serializers.py

```diff
@@ -215,12 +229,6 @@ def body_field_representation(page, request):
                             format_whatsapp_message(message, page, "whatsapp"),
                         ),
                         ("revision", page.get_latest_revision().id),
-                        ("is_whatsapp_template", page.is_whatsapp_template),
-                        ("whatsapp_template_name", page.whatsapp_template_name),
-                        (
-                            "whatsapp_template_category",
-                            page.whatsapp_template_category,
-                        ),
```

Comment:
I think in this API, we always returned these fields, even if it wasn't a template? e.g. a non-template message would be `is_whatsapp_template: False`.

In that case, to maintain backwards compatibility, we should always return these fields in this API, and just fill them with False/blank values if it's not a template message.